README.rst

```diff
@@ -297,6 +297,30 @@ since our editors can edit, add new content, and request a review for existing
 content. Therefore, it is not necessary to distinguish Editor and Contributor
 role.
 
+Transition-Guard expressions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Transition-guard expressions is a way to hide your transitions dynamically,
+in addition to the guard-roles. Use the options-syntax to define a guard-expression.
+
+Expressions in DCWorkflow are TALES expressions. To see the contextes provided
```

Comment:
`contextes` => `contexts`
and maybe `provided for expressions` => `available in expressions`?